tasties_app/tests/test_ingredient.py

```diff
@@ -0,0 +1,80 @@
+import pytest as pytest
+
+from tasties_app.models import Ingredient, Recipe, Category
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def ingredient():
+    """
+    Creates 2 users, 2 recipes, and an ingredient,
+    and adds them to the database.
+    """
+    user1 = User.objects.create_user('john', 'password')
+    user2 = User.objects.create_user('paul', 'password')
+    c1 = Category.objects.create(category_name='Breakfast')
+    c1.save()
+    test_data = [
+        ('Test Recipe1', user1, 'Test Description1', 'Test Directions1', 1, 'test_picture1'),
+        ('Test Recipe2', user2, 'Test Description2', 'Test Directions2', 2, 'test_picture2'),
+    ]
+    for title, aid, desc, dirs, mints, pic in test_data:
+        recipe = Recipe(title=title, author_id=aid, description=desc, directions=dirs,
+                        minutes_to_make=mints, recipe_picture=pic)
+        recipe.save()
+        recipe.categories.add(c1)
+        Ingredient(recipe_id=recipe, amount=1.0, measurement_unit='WHOLE',
+                   description='test ingredient').save()
+
+
+class TestIngredientModel:
+    """
+    Tests the Ingredient model:
+        - Creation of an ingredient
+        - Editing of an ingredient
+        - Negative tests to ensure that invalid data cannot be added to the database
+    """
+    @pytest.mark.django_db
+    def test_create_data_fixture(self, ingredient):
+        ingredient_set = Ingredient.objects.all()
+        ingredient1 = ingredient_set[0]
+        assert ingredient1.recipe_id.title == 'Test Recipe1'
+        assert ingredient1.description == 'test ingredient'
+        assert ingredient1.amount == 1.0
+        assert ingredient1.measurement_unit == 'WHOLE'
+        assert ingredient1.recipe_id.author_id.username == 'john'
+        assert ingredient1.recipe_id.categories.all()[0].category_name == 'Breakfast'
+        assert ingredient1 in Ingredient.objects.all()
+
+    @pytest.mark.django_db
+    def test_edit_ingredient(self, ingredient):
+        ingredient_set = Ingredient.objects.all()
+        ingredient1 = ingredient_set[0]
+        Ingredient.edit_info(ingredient1, 2.0, "Fluid Ounce", "test2")
+        assert ingredient1.amount == 2.0
+        assert ingredient1.measurement_unit == "Fluid Ounce"
+        assert ingredient1.description == "test2"
+
+    @pytest.mark.django_db
+    def test_invalid_amount_input(self, ingredient):
+        with pytest.raises(Exception):
+            recipe = Recipe.objects.all()[1]
+            ingredient2 = Ingredient.add(recipe, -3, "FLOZ", "test2")    # negative amount, should fail
```

Comment:
Please take a look at [pep8](https://peps.python.org/pep-0008/#inline-comments). Maybe you don't need these inline comments?